recipes/approvaltests.cpp/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+import os
+from conans import ConanFile, tools
+
+
+class ApprovalTestsCppConan(ConanFile):
+    name = "approvaltests.cpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/approvals/ApprovalTests.cpp"
+    license = "Apache-2.0"
+    description = "Approval Tests allow you to verify a chunk of output " \
+                  "(such as a file) in one operation as opposed to writing " \
+                  "test assertions for each element."
+    topics = ("conan", "testing", "unit-testing", "header-only")
+    options = {
+        "with_catch2": [True, False],
+        "with_gtest": [True, False],
+        "with_doctest": [True, False]
+    }
+    default_options = {
+        "with_catch2": False,
+        "with_gtest": False,
+        "with_doctest": False
+    }
+    no_copy_source = True
+
+    @property
+    def _header_file(self):
+        return "ApprovalTests.hpp"
+
+    def requirements(self):
+        if self.options.with_catch2:
+            self.requires("catch2/2.11.0")
+        if self.options.with_gtest:
+            self.requires("gtest/1.10.0")
+        if self.options.with_doctest:
+            self.requires("doctest/2.3.6")
+
+    def source(self):
+        for source in self.conan_data["sources"][self.version]:
+            url = source["url"]
+            filename = url[url.rfind("/") + 1:]
+            tools.download(url, filename)
+            tools.check_sha256(filename, source["sha256"])
+        os.rename("ApprovalTests.v.{}.hpp".format(self.version),
+                  self._header_file)
+
+    def package(self):
+        self.copy(self._header_file, dst="include", src=self.source_folder)
```

Comment:
`src=self.source_folder` part is not necessary (see bullet points at the end of description of the [package() method](https://docs.conan.io/en/latest/reference/conanfile/methods.html#package))

Reply:
Done